File: Sorting/Insertion Sort/InsertionSort.php
***** PHP program *****
 <html>
  <head>
   <title>
    Insertion sort
   </title>
  </head>
  <body>
   <?php
    $a = array(10,9,7,101,23,44,12,78,34,23);
    echo("printing sorted elements...\n");
    for($k=0;$k<10;$k++)
    {
     $temp = $a[$k];
     $j = $k-1;
     while($j>=0 && $temp<=$a[$j])
     {
      $a[$j+1] = $a[$j];
      $j = $j-1;
     }
     $a[$j+1] = $temp;
    }
    for($i=0;$i<10;$i++)
    { 
     echo $a[$i];
     echo "\n";
    }
   ?>
  </body>
 </html>

File: Sorting/Insertion Sort/InsertionSort.swift
***** Swift program *****
import Foundation
import Glibc
 var a = [10,9,7,101,23,44,12,78,34,23];
 print("\n printing sorted elements... \n");
 for k in 1...9
 {
  let temp = a[k];
  var j=k-1;
  while j>=0 && temp<=a[j]
  {
   a[j+1] = a[j];
   j = j-1;
  }
  a[j+1] = temp;
 }
 for i in a
 {
  print(i);
 }
